data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+	description `Scan the cargo of the Hai Geocoris in Ya Hai with the Unfettered scanner, then return to <destination> with the scanner logs.`
+	source "Firelode"
+	waypoint "Ya Hai"
+	to offer
+		has "Unfettered: Bring Medical Supplies: done"
+		has "Unfettered: Prove Your Strength: done"
+		not "Wanderers: Defend Vara Ke'sok: done"
+	on offer
+		conversation
+			`As you wander through the spaceport, you find a market. You notice that the amount and diversity of goods are very limited. Some of it seems to be completely alien, even by Hai standards. Most Unfettered here are just busy buying food, that is if they can even afford it. You see two Unfettered disputing, probably over the cost of some item, but decide not to intervene given how they generally respond to your mere presence...`
+			`	An Unfettered approaches you. At first you wonder why the Hai happens to be coming towards you in particular, but then you realize that you naturally stand out here. There are not many non-Hai on the Unfettered worlds.`
+			`	"Captain <last>! I am Taer-Fi, an engineer. We have an easy job for you, if you are interested," she says.`
+			choice
+				`	"Sure, what can I do for you?"`
+				`	"How do you know me?"`
+					goto know
+				`	"Sorry, but I'm quite busy at the moment."`
+					defer
+				`	"I don't want to help the Unfettered in any way."`
+					decline
+			`	"Straight to the point! I am starting to like you."`
+				goto job
+			label know
+			`	"Someone who brings us something as precious as a jump drive is not quickly forgotten! Now back to our business, if you don't mind."`
+			label job
+			`	"As you probably already know, the other Hai do not think like us. They believe in trading with all the galaxy in harmony forever, without seeing how it softens them. As such, they have wasted time designing a ship for the sole purpose of transporting goods in great quantities: the Geocoris. Or at least that is what we thought, at first. It now seems to us that the design is... interesting, so we want you to scan it."`
+			`	"It is pretty much guaranteed there will be one such ship at all times around their home system, Ya-Hai, the center of their trade. For obvious reasons we cannot scan that ship ourselves, and we prefer that the Hai be ignorant of our interest in that design, anyway. Bring us the full scanner logs and you will be well rewarded."`
+			choice
+				`	"Seems easy. I'll do it."`
+					goto agree
+				`	"Why the sudden interest for that ship?"`
+					goto design
+				`	"You could just ask the Hai that come to you from their space about it."`
+				`	"I don't want to get in trouble with the Hai."`
+					decline
+			`	"That is what we will be forced to do if you refuse, but we would prefer this to be kept a secret. Also most of the Hai who join us are not warriors, but rather artists, others who want to explore diverse ways of seeing the world, or even simply disagree with the ways of the Elders. So do you agree to do this for us?"`
```

Comment:
It should be moved to another place indeed

This elder will want to break the false ideas you probably have about the unfettered, because of propaganda and appearances

Reply:
Gotcha. Let's find another place where it fits better then.